README.md

```diff
@@ -86,6 +86,7 @@ conn.execute("INSERT INTO friend (name) VALUES (?1)", params!["John"])
 Please see the [examples](https://github.com/cljoly/rusqlite_migrate/tree/master/examples) folder for more, in particular:
 - `async` migrations in the [`quick_start_async.rs`][quick_start_async] file
 - migrations with multiple SQL statements (using for instance `r#"…"` or `include_str!(…)`)
+- loading migrations from files
```

Comment:
Note for myself: I thought I added that already, check that.

Reply:
I actually had these changes locally, opened a PR now: https://github.com/cljoly/rusqlite_migration/pull/103/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R41

Since this is only tangentially related to the title on this PR, I would suggest keeping it just #103. Feel free to comment there if you would like.